src/current/v24.3/schema-design-overview.md

```diff
@@ -120,12 +120,18 @@ The following table lists specific limits imposed by CockroachDB.
 
 ### Quantity of tables and other schema objects
 
-CockroachDB has been shown to perform well with clusters containing 2,500 tables. Greater numbers are possible, depending on the complexity of the tables (number of columns and indexes) and hardware specifications.
+CockroachDB has been shown to perform well with clusters containing the following numbers of schema objects:
+
+- 25,000 tables for single region clusters.
+- 10,000 tables for [multi-region clusters]({% link {{ page.version.version }}/multiregion-overview.md %}).
+
+Greater numbers are possible, depending on the complexity of the tables (number of columns and indexes) and hardware specifications.
 
 As you scale to a large number of tables, note that:
 
 - The amount of RAM per node is the limiting factor for the number of tables and other schema objects the cluster can support. This includes columns, indexes, GIN indexes, constraints, and partitions. Increasing RAM is likely to have the greatest impact on the number of these objects that a cluster can support, while increasing the number of nodes will not have a substantial effect.
 - The number of databases or schemas on the cluster has minimal impact on the total number of tables that it can support.
+- Performance at larger numbers of tables may be affected by your use of [Backups]({% link {{ page.version.version }}/backup-and-restore-overview.md %}) and [Change Data Capture]({% link {{ page.version.version }}/change-data-capture-overview.md %}).
```

Comment:
IIRC it was restore that would be affected more so than backups. @alicia-l2 can confirm!

Reply:
Yes maybe we could just say Backup/Restore

Reply:
for now i will update to say "backup and restore" to keep it vague.  Alicia, happy to update next week with your feedback, rushing to get this in for Monday's release at the moment :-)

Reply:
omg @alicia-l2 i totally missed your comment but am seeing it now, sorry about that - maybe i needed to refresh the page

luckily i wrote "backup and restore" :-)